src/pages/Popup/index.tsx

```diff
@@ -26,6 +26,10 @@ const renderApp = async () => {
     buster: VERSION,
   });
   const container = document.getElementById('app');
+
+  // We need to do this after load in order for the extension popup to have the correct height on first load
+  container!.style.maxHeight = '100vh';
+
```

Comment:
@victorkirov this was part of another PR right? maybe we can add a note on the merge order to make this PR easier to review?

Reply:
Ah, yes. Sorry, will add that 👍

Reply:
thanks! actually, I guess it doesn't help a whole lot 😂